### ISSUE TITLE
Please bump axios version to get rid off npm warning

### ISSUE THREAD

**Issue by muscle-rhnkyr** [ratiw/vuetable-2#624]

**Comment by azzhcoder** [comment]
Yeesss, as sooon as possible!

**Comment by lionslair** [comment]
Created a pull request for it.

**Comment by stelgenhof** [comment]
Yes, please upgrade the Axios dependency as soon as possible. Axios has a security vulnerability which has been addressed already (See also: https://www.npmjs.com/advisories/880)

Thanks!

**Comment by irufus** [comment]
Any updates for this issue?

**Comment by lionslair** [comment]
This was my pr https://github.com/ratiw/vuetable-2/pull/625

**Comment by irufus** [comment]
Looks like this was addressed in pr #183 to turn into a peer dependency but it hasn't been merged.

**Comment by lionslair** [comment]
Big gap between merged prs
![Selection_003](https://user-images.githubusercontent.com/832259/73918720-1d140b00-48fd-11ea-8726-b3ae9fd60a35.png)

**Comment by irufus** [comment]
and now I have created a fork for my corp.
